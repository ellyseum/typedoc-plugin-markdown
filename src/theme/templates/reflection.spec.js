const context = {
  project: {
    name: 'typedoc-plugin-markdown',
  },
  model: {
    kindString: 'External module',
    name: 'Reflection name',
    hasComment: true,
    typeHierarchy: {},
    typeParameters: {},
    implementedTypes: [0, 0],
    implementedBy: [0, 0],
    signatures: {},
    indexSignature: {},
    args: {},
  },
};

describe(`reflection.hbs`, () => {
  test('should compile', () => {
    const result = global.compileTemplate('reflection.hbs', context, 'template');
    expect(result).toMatchSnapshot();
  });
});
