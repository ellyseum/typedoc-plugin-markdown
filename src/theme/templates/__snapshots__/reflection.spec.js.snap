// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reflection.hbs should compile 1`] = `
"[typedoc-plugin-markdown](../README.md)[breadcrumb]

# External module: Reflection name

[comment]

## Type parameters

[typeParameters]

## Hierarchy

[hierarchy]

## Implements

* [type]
* [type]

## Implemented by

* [type]
* [type]

## Callable

[member.signatures]

## Indexable

[member.indexSignatures]

[index]
[members]"
`;
